examples/server-communication__xhr-assertions/cypress/e2e/multiple-requests.cy.js

```diff
@@ -19,22 +18,22 @@ describe('waits', () => {
     cy.get('@post.all').should('have.length', 2)
     .then((xhrs) => {
     // xhrs is an array of network call objects
-      expect(xhrs[0], 'first request status').to.have.property('status', 201)
-      expect(xhrs[1], 'second request status').to.have.property('status', 201)
+      expect(xhrs[0].response, 'first request status').to.have.property('statusCode', 201)
+      expect(xhrs[1].response, 'second request status').to.have.property('statusCode', 201)
     })
 
     // and we can make assertions about each separate call
     // by retrieving it like this (index starts with 1)
     // cy.get('<alias>.<index>')
     cy.get('@post.1').should((xhr1) => {
-      expect(xhr1, 'first request').to.have.property('status', 201)
+      expect(xhr1.response, 'first request').to.have.property('statusCode', 201)
     })
 
     // we cannot guarantee the order of XHR requests - they really depend on the
     // server response speed. Sometimes a later request finishes first.
     // all we can say that each request should receive a response with
     // id equal to 101 or 102
-    cy.get('@post.1').its('response.body.id').should('be.oneOf', [101, 102])
-    cy.get('@post.2').its('response.body.id').should('be.oneOf', [101, 102])
+    cy.get('@post.1').its('response').its('body.id').should('be.oneOf', [101, 102])
```

Comment:
Can these be `.its('response.body.id')`?

Reply:
They sure can.